app/controllers/api/groups_controller.rb

```diff
@@ -161,6 +161,46 @@ def update_marks
         HttpStatusHelper::ERROR_CODE['message']['200'] }, status: 200
     end
 
+    def update_extra_marks
```

Comment:
"update" implies that we're changing an existing extra mark. In this case we're creating an extra mark. We should also have the ability to remove extra marks (probably as another route).